roles/rke2_common/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 - name: SLES/Ubuntu/Tarball Installation
   include: tarball_install.yml
   when:
-    - ansible_facts['os_family'] != 'RedHat' or rke2_binary_tarball_check.stat.exists == true
+    - ansible_facts['os_family'] != 'RedHat' or ansible_facts['os_family'] != 'Rocky' or rke2_binary_tarball_check.stat.exists == true
```

Comment:
Same as the above comment https://github.com/rancherfederal/rke2-ansible/pull/64/files#r682706026

Reply:
Fixed too.